Neos.ContentRepository.Core/Classes/Projection/ContentStream/ContentStreamProjection.php

```diff
@@ -379,7 +377,7 @@ private function updateStateForContentStream(ContentStreamId $contentStreamId, C
     }
 
     private function updateContentStreamVersion(
-        EmbedsContentStreamAndNodeAggregateId $eventInstance,
+        EmbedsContentStreamId $eventInstance,
```

Comment:
```suggestion
        EmbedsContentStreamId&EmbedsNodeAggregateId $eventInstance,
```
would be the "old" behavior – but maybe the new one is more correct!?